saleor/registration/forms.py

```diff
@@ -19,6 +21,18 @@ def __init__(self, request=None, *args, **kwargs):
             if email:
                 self.fields['username'].initial = email
 
+    def confirm_login_allowed(self, user):
+        super().confirm_login_allowed(user)
+
+        if (settings.EMAIL_VERIFICATION_REQUIRED and not user.is_staff
+            and not user.email_verified):
+            send_activation_mail(self.request, user)
```

Comment:
I think it's not a really good idea to resend the mail on login. Maybe you could replace it with a cooldown or only allow to resend the mail once. Plus, I think it would be better if the user had to click on a resend button instead of sending it automatically.